presto/presto.go

```diff
@@ -585,6 +587,16 @@ type stmtStage struct {
 	SubStages       []stmtStage `json:"subStages"`
 }
 
+// ErrQueryId indicates that a query is success, only for give QueryId.
+type ErrQueryId struct {
```

Comment:
By reading the rest of the code it seems this error is a wrapper for io.EOF that can carry over the QueryID that failed. Could we please rename it to EOF instead of ErrQueryId?

Reply:
thank for your review, I have modified it as you suggested